index.js

```diff
@@ -62,24 +62,16 @@ module.exports = function (options) {
 			if (errorList.length > 0) {
 				file.jscs.success = false;
 				file.jscs.errorCount = errorList.length;
-				file.jscs.errors = errorList;
+				file.jscs.errors = errors;
 			}
-
-			errorList.forEach(function (err) {
-				out.push(errors.explainError(err, true));
-			});
 		} catch (err) {
-			out.push(err.stack.replace('null:', file.relative + ':'));
+			console.error(err.stack.replace('null:', file.relative + ':'));
 		}
 
 		cb(null, file);
-	}, function (cb) {
-		if (out.length > 0) {
-			this.emit('error', new gutil.PluginError('gulp-jscs', out.join('\n\n'), {
-				showStack: false
-			}));
-		}
-
-		cb();
 	});
 };
+
+exports.failReporter = reporters.fail;
```

Comment:
JSHint was exposing it, so I did as well. But yes, this is unnecessary, removed.